public/js/views/management/pay-methods.jsx

```diff
@@ -20,6 +21,10 @@ export default class PayMethods extends Component {
 
   handleDelPayMethod = e => {
     e.preventDefault();
+    if (isDisabled(e.target)) {
```

Comment:
this seems unfortunate because I feel like one day someone is going to forget to check for this. Is there really no react way to delete an element so that its event handlers aren't fired?